tests/queries/0_stateless/01595_countMatches.sql

```diff
@@ -0,0 +1,29 @@
+select 'basic';
+select countMatches('', 'foo');
+select countMatches('foo', '');
+-- simply stop if zero bytes was processed
+select countMatches('foo', '[f]{0}');
```

Comment:
Note, that current implementation will simply stop if zero bytes was processed, however this looks pretty synthetic so this should not be a big deal, thoughts?

Reply:
I think this is OK.